pipeline-braille-scripts/epub3-to-pef/src/main/resources/css/default.scss

```diff
@@ -0,0 +1,97 @@
+/*
```

Comment:
Yeah, but on the other hand these are format-specific rules. It's another reason to merge the modules maybe?